exercises/parallel-letter-frequency/examples/success-standard/package.yaml

```diff
@@ -1,4 +1,5 @@
 name: parallel-letter-frequency
+version: 0.1.0.3
```

Comment:
I recommend not adding it. You never know what might look upon this as an example. So we should be consistent if we're not adding it.